release-1.9/release_team.md

```diff
@@ -0,0 +1,13 @@
+|  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
```

Comment:
do we use the shadow names here?

Reply:
Why not?

Reply:
I wasn't saying we shouldn't have shadows; I was asking whether we really have/are planning to have shadows?